mangum/protocols/lifespan.py

```diff
@@ -63,7 +62,7 @@ class LifespanCycle:
     def __post_init__(self) -> None:
         self.logger = logging.getLogger("mangum.lifespan")
         self.loop = asyncio.get_event_loop()
-        self.app_queue: asyncio.Queue = asyncio.Queue()
+        self.app_queue: asyncio.Queue[typing.Dict[str, str]] = asyncio.Queue()
```

Comment:
What about if instead this uses the `Message` type alias in https://github.com/jordaneremieff/mangum/blob/main/mangum/types.py#L17?